### PR TITLE
add log redaction-related contents

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

add log redaction-related contents

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5186
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [x] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: log-redaction.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+From the error log above, you can see that all sensitive information is shielded using `?` after `tidb_redact_log` is enabled. In this way, data security risks are avoided.
+
+## Log redaction in TiKV side
+
+To enable log redaction in the TiKV side, set the value of [`security.redact-info-log`](/tikv-configuration-file.md#redact-info-log-new-in-v408) to `true`. This configuration value defaults to `false`, which means that log redaction is disabled.
+
+## Log redaction in PD side
+
+To enable log redaction in the PD side, set the value of [`security.redact-info-log`](/pd-configuration-file.md#redact-info-log-new-in-v408) to `true`. This configuration value defaults to `false`, which means that log redaction is disabled.
```
Comment (TomShawn) [inline log-redaction.md:45]:
```suggestion
To enable log redaction in the PD side, set the value of [`security.redact-info-log`](/pd-configuration-file.md#redact-info-log-new-in-v500-rc) to `true`. This configuration value defaults to `false`, which means that log redaction is disabled.
```

File: log-redaction.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+```sql
+set @@global.tidb_redact_log=1;
+```
+
+After the setting, all logs generated in new sessions are redacted:
+
+```sql
+create table t (a int, unique key (a));
+Query OK, 0 rows affected (0.00 sec)
+insert into t values (1),(1);
```
Comment (ran-huang) [inline log-redaction.md:27]:
```suggestion

insert into t values (1),(1);
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: pd-configuration-file.md [hunk 3]
```diff
@@ -143,6 +143,12 @@ Configuration items related to security
 + The path of the PEM file that contains the X509 key
 + Default value: ""
 
+### `redact-info-log` <span class="version-mark">New in v5.0.0-rc</span>
```
Comment (TomShawn) [inline pd-configuration-file.md:146]:
```suggestion
### `redact-info-log` <!-- New in v5.0.0-rc -->
```

File: log-redaction.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+From the error log above, you can see that all sensitive information is shielded using `?` after `tidb_redact_log` is enabled. In this way, data security risks are avoided.
+
+## Log redaction in TiKV side
+
+To enable log redaction in the TiKV side, set the value of [`security.redact-info-log`](/tikv-configuration-file.md#redact-info-log-new-in-v408) to `true`. This configuration value defaults to `false`, which means that log redaction is disabled.
+
+## Log redaction in PD side
+
+To enable log redaction in the PD side, set the value of [`security.redact-info-log`](/pd-configuration-file.md#redact-info-log-new-in-v500-rc) to `true`. This configuration value defaults to `false`, which means that log redaction is disabled.
```
Comment (TomShawn) [inline log-redaction.md:46]:
```suggestion
To enable log redaction in the PD side, set the value of [`security.redact-info-log`](/pd-configuration-file.md#redact-info-log) to `true`. This configuration value defaults to `false`, which means that log redaction is disabled.
```

**Pull request merged**